### ISSUE TITLE
 No value found for 'CreateAliasTransaction'

### ISSUE THREAD

**Issue by jimisan** [wavesplatform/Waves#332]
java -jar waves-0.6.5.jar  ~/.waves/waves.conf 2>&1 | tee ~/waves.log
2017-05-31 11:46:35 INFO  [main] c.w.Application$ - Starting...
Exception in thread "main" java.util.NoSuchElementException: No value found for 'CreateAliasTransaction'
        at scala.Enumeration.$anonfun$withName$2(Enumeration.scala:125)
        at scala.Enumeration.withName(Enumeration.scala:125)
        at com.wavesplatform.settings.FeesSettings$.toTxType(FeesSettings.scala:20)
        at com.wavesplatform.settings.FeesSettings$.toFeeSettings(FeesSettings.scala:36)
        at com.wavesplatform.settings.FeesSettings$.$anonfun$fromConfig$2(FeesSettings.scala:25)
        at scala.collection.TraversableOnce.$anonfun$foldLeft$1(TraversableOnce.scala:157)
        at scala.collection.TraversableOnce.$anonfun$foldLeft$1$adapted(TraversableOnce.scala:157)
        at scala.collection.Iterator.foreach(Iterator.scala:929)
        at scala.collection.Iterator.foreach$(Iterator.scala:929)
        at scala.collection.AbstractIterator.foreach(Iterator.scala:1406)
        at scala.collection.IterableLike.foreach(IterableLike.scala:71)
        at scala.collection.IterableLike.foreach$(IterableLike.scala:70)
        at scala.collection.AbstractIterable.foreach(Iterable.scala:54)
        at scala.collection.TraversableOnce.foldLeft(TraversableOnce.scala:157)
        at scala.collection.TraversableOnce.foldLeft$(TraversableOnce.scala:155)
        at scala.collection.AbstractTraversable.foldLeft(Traversable.scala:104)
        at com.wavesplatform.settings.FeesSettings$.fromConfig(FeesSettings.scala:24)
        at com.wavesplatform.settings.WavesSettings$.fromConfig(WavesSettings.scala:38)
        at com.wavesplatform.Application$.main(Application.scala:129)
        at com.wavesplatform.Application.main(Application.scala)

**Comment by alexeykiselev** [comment]
Jim, you somehow mixed the 0.6 and 0.7 versions. CreateAliasTransaction will be available only in 0.7.

**Comment by jimisan** [comment]
What exactly I mixed? 0.7 was compiled on monday, deleted now. 0.6 was compiled today few moments ago. What I need to do next?

**Comment by jimisan** [comment]
It's looks like I'm done. Thank you.

**Comment by willinspire** [comment]
I also had the same issue. 
`Exception in thread "main" java.util.NoSuchElementException: No value found for 'CreateAliasTransaction'
	at scala.Enumeration.$anonfun$withName$2(Enumeration.scala:125)
	at scala.Enumeration.withName(Enumeration.scala:125)
	at com.wavesplatform.settings.FeesSettings$.toTxType(FeesSettings.scala:20)
	at com.wavesplatform.settings.FeesSettings$.toFeeSettings(FeesSettings.scala:36)
	at com.wavesplatform.settings.FeesSettings$.$anonfun$fromConfig$2(FeesSettings.scala:25)
	at scala.collection.TraversableOnce.$anonfun$foldLeft$1(TraversableOnce.scala:157)
	at scala.collection.TraversableOnce.$anonfun$foldLeft$1$adapted(TraversableOnce.scala:157)
	at scala.collection.Iterator.foreach(Iterator.scala:929)
	at scala.collection.Iterator.foreach$(Iterator.scala:929)
	at scala.collection.AbstractIterator.foreach(Iterator.scala:1406)
	at scala.collection.IterableLike.foreach(IterableLike.scala:71)
	at scala.collection.IterableLike.foreach$(IterableLike.scala:70)
	at scala.collection.AbstractIterable.foreach(Iterable.scala:54)
	at scala.collection.TraversableOnce.foldLeft(TraversableOnce.scala:157)
	at scala.collection.TraversableOnce.foldLeft$(TraversableOnce.scala:155)
	at scala.collection.AbstractTraversable.foldLeft(Traversable.scala:104)
	at com.wavesplatform.settings.FeesSettings$.fromConfig(FeesSettings.scala:24)
	at com.wavesplatform.settings.WavesSettings$.fromConfig(WavesSettings.scala:38)
	at com.wavesplatform.Application$.main(Application.scala:129)
	at com.wavesplatform.Application.main(Application.scala)`
I'm unsure how this error of version-mixing occurred when I only installed the latest version as per the instructions provided. I'll look for the CreateAliasTransaction to remove it. Any assistance as to where this item is located would be much appreciated. 

Thank you.

**Comment by willinspire** [comment]
Okay - I think that I've discovered the problem. The recommended **waves.conf** configurations include the 0.7 parameters for this "CreateAlies" in the "Transaction" configuration settings. When users install the Waves node using recommended settings then all will go well unless they decide to use the recommended settings for **waves.conf**. Perhaps a note under that section would be appropriate so as to inform new users of this minor issue (just a thought). 

After the alias transaction configuration was removed from **waves.conf** the node can be started without any problems.
